content/news/intrapixel-detector-SPIE.md

```diff
@@ -0,0 +1,20 @@
+Title: Intrapixel Detector Responsivity Measured on Kepler Flight Spare
```

Comment:
Blog post titles are not capitalized (by tradition). I suggest `Intrapixel responsivity measured on a Kepler flight spare detector`